src/components/SidebarTabs/SharingSidebarTab.vue

```diff
@@ -453,5 +488,11 @@ export default {
 		margin-inline-end: 4px;
 		color: var(--color-error)
 	}
+
+}
```

Comment:
```suggestion
}
```
The empty line is still there... ;)